patch-test.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-env node, es6 */
+/* eslint-disable no-console */
+/* eslint-disable es5/no-destructuring */
+/* eslint-disable es5/no-template-literals */
+/* eslint-disable es5/no-es6-methods */
+/* eslint-disable es5/no-shorthand-properties */
+/* eslint-disable es5/no-block-scoping */
```

Comment:
Ugly. The `eslint-env` setting ought to automatically disable these es5 rules. Reported issue upstream: https://github.com/nkt/eslint-plugin-es5/issues/40